src/tespy/components/turbomachinery.py

```diff
@@ -1269,6 +1269,32 @@ def calc_parameters(self):
 
         self.check_parameter_bounds()
 
+    def exergy_balance(self, Tamb_val_SI, bus):
+        r"""
+        Calculate exergy balance of a pump.
+
+        Parameters
+        ----------
+        Tamb_val_SI : float
+            Ambient temperature in K.
+        bus  : tespy.connections.bus
+            Energy flows in network. Used to calculate product exergy
+            or fuel exergy of turbines and pumps.
+            
+        Note
+        ----
+            .. math::
+
+                E_{P} = \dot{m}_{in} \cdot (e_{ph,out} - e_{ph,in})
+                E_{F} = P
+        """
+        self.E_P = self.inl[0].m.val_SI * (self.outl[0].ex_physical
+                                                 - self.inl[0].ex_physical)
+        if bus == None:
+            self.E_F = self.P.val
+        else:
+            self.E_F = self.calc_bus_value(bus)
+
```

Comment:
This is, so instead of the pump power the exergy input can be the electrical power of the motor?

Reply:
Yes
If the energy flow is not in a bus, there is no electrical efficiency of the pump. The pump power is used for fuel exergy (=input exergy). If we have an electrical efficiency, the fuel exergy is the electrical power of the motor.